src/task3/config/database.js

```diff
@@ -0,0 +1,15 @@
+const { Sequelize } = require('sequelize');
+
+export const db = new Sequelize('nodeMP', 'postgres', '123456', {
+    host: 'localhost',
+    dialect: 'postgres',
+    operatorsAliases: false,
+    pool: {
+        max: 5,
+        min: 0,
+        acquire: 30000,
+        idle: 10000
+    }
+});
+
+// const sequelize = new Sequelize('postgres://postgres:123456@127.0.0.1:5432/nodeMP');
```

Comment:
It would be better to have opportunity to override DB setting according ENV (local, QA, PROD) ;

```
const env = process.env.NODE_ENV || 'development';
cons config    = require('../config/config.json')[env];
```

here is a sample of such config
```
{
  "development": {
    "username": "root",
    "password": "root",
    "database": "database name",
    "host": "127.0.0.1",
    "dialect": "mysql"
  },
  "test": {
    "username": "root",
    "password": "root",
    "database": "database name",
    "host": "127.0.0.1",
    "dialect": "mysql"
  },
  "production": {
    "username": "root",
    "password": "root",
    "database": "database name",
    "host": "127.0.0.1",
    "dialect": "mysql"
  }
}
```

Reply:
Also it's good practice to configure such kind of setting via ENV variables (not necessary to implement), you can just a read about it.

```
"start:dev": "cross-env NODE_ENV=development DB_HOST=dbhost etc... node app.js",
```

And then access this data in code in such way
```
process.env.DB_HOST
```